test/inject-test-mode

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# This script injects a "test-mode" file into a copy of the main disk image. 
+# It begins by parsing the partition table of the disk image to identify the
+# 'aux' partition, which is then extracted. An empty 'test-mode' file is 
+# injected into the extracted partition, and the modified partition is written
+# back to the output image. The output image can subsequently be used as a 
+# backing image for Copy-on-Write (QCOW) images utilized by Qeneth.
+
+while getopts "b:o:" opt; do
+  case $opt in
+    b) base_img="$OPTARG" ;;  # Base image (Original image)
+    o) output_img="$OPTARG" ;;  # Output image (Backing image for QCoW images)
+    *) echo "Usage: $0 -b <BASE-IMAGE> -o <OUTPUT-IMAGE>" ; exit 1 ;;
+  esac
+done
+
+if [ -z "$base_img" ] || [ -z "$output_img" ]; then
+  echo "Both -b (base image) and -o (output image) parameters are required."
+  exit 1
+fi
+
+rm -f "$output_img"
+cp "$base_img" "$output_img"
+
+part_table=$(fdisk -l "$output_img")
+aux_line=$(echo "$part_table" | grep 'aux')
+start=$(echo "$aux_line" | awk '{print $2}')
+end=$(echo "$aux_line" | awk '{print $3}')
+count=$(($end - $start + 1))
+block_size=$(echo "$part_table" | grep "Logical sector size" | awk '{print $4}')
+
+dd if="$output_img" of="tmpaux" skip="$start" count="$count" bs="$block_size" > /dev/null 2>&1
```

Comment:
```suggestion
dd if="$output_img" of="tmpaux" skip="$start" count="$count" bs="$block_size" status=none
```
This will turn of the statistics report at the end (`X+0 records in/out`), but still print any errors such as missing files etc.

Reply:
Ahaa, I see. I'll fix it :) Thnak you